test_scripts/API/PixelDensityAndScale/004_Get_VideoStreamCapability_only_with_scale_SUCCESS.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0179-pixel-density-and-scale.md
+-- Description:
+-- In case:
+-- 1) Update hmiCapabilities for videoStreamingCapability param (scale)
+-- 2) Mob app sends GetSystemCapability request to SDL
+-- SDL does:
+-- 1) Send response to Mobile only with scale param for videoStreamingCapability
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/PixelDensityAndScale/commonPixelDensity')
+local hmi_values = require('user_modules/hmi_values')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local hmiValues = hmi_values.getDefaultHMITable()
+
+local diagonalScreenSize = nil
+local pixelPerInch = nil
+local scale = 1.05
+
+local function updateHMIValue(pDiagonalSize, pPixelPerInch, pScale)
+    hmiValues.UI.GetCapabilities.params.systemCapabilities.videoStreamingCapability.diagonalScreenSize = pDiagonalSize
+    hmiValues.UI.GetCapabilities.params.systemCapabilities.videoStreamingCapability.pixelPerInch = pPixelPerInch
+    hmiValues.UI.GetCapabilities.params.systemCapabilities.videoStreamingCapability.scale = pScale
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Update HMI Capabilities", updateHMIValue, { diagonalScreenSize, pixelPerInch, scale })
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start, { hmiValues })
+runner.Step("RAI", common.registerApp)
+runner.Step("Activate App", common.activateApp)
+
+-- [[ Test ]]
+runner.Title("Test")
+runner.Step("Get Capability", common.getSystemCapability, { nil, nil, scale })
```

Comment:
In case some values are not provided, must be added the check that they are absent.